pkg/runctl/controller_test.go

```diff
@@ -229,6 +229,7 @@ func Test_Controller_syncHandler_delete(t *testing.T) {
 }
 
 func Test_Controller_syncHandler_mock(t *testing.T) {
+	expectedError := fmt.Errorf("expected")
```

Comment:
There are still occurrences of the extracted expression. Replace those by this variable.

Reply:
This is by intention. I only have expectedError used where I do a compare of the returned error.
If I check the error message I use the fmt.Error near the test case for better readability.

Reply:
Renamed to error1 and use variable everywhere.